examples/c/ex_backup_block.c

```diff
@@ -310,7 +312,7 @@ take_incr_backup(WT_SESSION *session, int i)
     WT_CURSOR *backup_cur, *incr_cur;
     uint64_t offset, size, type;
     size_t alloc, count, rdsize, tmp_sz;
-    int j, ret, rfd, wfd;
+    int j, range, ret, rfd, wfd;
```

Comment:
Nitpick: Maybe use a better name that aligns with the purpose of counting the number of ranges, e.g., `nranges` as in the incremental backup example below? But this is fine too.

Reply:
Fixed.